packages/next/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@makeswift/next",
+  "version": "0.0.0",
+  "license": "MIT",
+  "repository": {
+    "url": "makeswift/makeswift",
+    "directory": "packages/next"
+  },
+  "sideEffects": false,
+  "main": "src/index.ts",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.js",
+    "module": "dist/index.mjs",
+    "typings": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "tsup"
+  },
+  "peerDependencies": {
+    "next": "^13.4.5"
```

Comment:
We should add `react` and `react-dom` as peer (and dev) dependencies as well

Reply:
Good point. I was going to add them when we needed to use the dependencies in code, but I guess this would be good to add now.

Reply:
I also added corresponding `@types/**` 🙏

Reply:
Yeah, it came to mind since Next.js itself has those as peer dependencies. Are `@types/*` usually peer deps as well? We might want to make those optional peer deps